Documentation/Minor/Index.rst

```diff
@@ -28,14 +28,14 @@ There are two ways to check if a patch/bugfix update is available for an install
 
 All supported versions of TYPO3 and their version numbers are published on `get.typo3.org <https://get.typo3.org>`_.
 
-Alternatively, running `composer outdated -m "typo3/*"` will present a list of any TYPO3 packages that have patch/bugfix updates.
+Alternatively, running  :bash:`composer outdated -m "typo3/*"` will present a list of any TYPO3 packages that have patch/bugfix updates.
 
 Execute the update
 ==================
 
-To execute the update, run `composer update --with-all-dependencies "typo3/*"`.
+To execute the update, run :bash:`composer update --with-all-dependencies "typo3/*"`.
 
-This will update all TYPO3 packages, the `--with-all-dependencies` signals that any dependencies of TYPO3 should also be updated.
+This will update all TYPO3 packages, the :bash:`--with-all-dependencies` signals that any dependencies of TYPO3 should also be updated.
```

Comment:
This is not correct like written.  
A simple `composer update` will update everything, no matter about dependencies to each other.  
`composer update --with-all-dependencies "typo3/*"` in contrast will take care on dependencies between TYPO3 packages and their dependencies. This means that some dependencies might not get updated to most recent versions perhaps, if TYPO3 uses and older version.

Reply:
@DavidBruchmann : Please open another PR for this after this one has been closed!

Reply:
```suggestion
This will update all TYPO3 packages, the :bash:`--with-all-dependencies` signals that any dependencies of TYPO3 should also be updated with respect to the version constraints given by the TYPO3 packages.
```

Reply:
#388

Reply:
Please:
This PR is only for the Bash-command and nothing else!